di-ipv-ticf-stub/lambdas/jest.config.test.js

```diff
@@ -4,8 +4,6 @@ module.exports = {
   coverageDirectory: "coverage",
   coverageProvider: "v8",
 
-  preset: "@shelf/jest-dynamodb",
-
```

Comment:
This was preventing me from running the unit tests locally and I think we stub methods which make calls to dynamodb anyway so not sure why we'd need this jest preset